docs/scripts/formattedTSDemos.js

```diff
@@ -144,6 +145,18 @@ async function main(argv) {
     ...(await getFiles(path.join(workspaceRoot, 'docs/data'))), // new structure
   ];
 
+  const paths = await getFiles(path.join(workspaceRoot, 'docs/data'));
+  const transformPath = path.resolve('scripts/addDemoItemsAttributes.js');
+
+  // Format pickers demos such that `DemoContainer` and `DemoItem` gets correct `components` prop
+  await jscodeshift(transformPath, paths, {
+    extensions: 'js,ts,jsx,tsx',
+    parser: 'tsx',
+    verbose: 2,
+    quote: 'single',
+    trailingComma: true,
```

Comment:
```suggestion
    trailingComma: true,
    wrapColumn: 85
```
The default of `74` is way to "opinionated" 🙈 
Maybe use a value of our `prettier`? 🤔

Reply:
I tried lot of parameters, but the prettier always modify something after

Reply:
To avoid problems, I moved the codemod script to a distinct command, such that everybody is free to run it or not, and avoid CI problems

I also opened follow-up issue #7691

I'm not sure this codemod will survive to the resolution of the follow-up issue, and will be happy to delete this command when solved :)